packages/website/ts/utils/order_parser.ts

```diff
@@ -0,0 +1,33 @@
+import { logUtils } from '@0xproject/utils';
+import * as _ from 'lodash';
+
+import { portalOrderSchema } from 'ts/schemas/portal_order_schema';
+import { validator } from 'ts/schemas/validator';
+import { Order } from 'ts/types';
+
+export const orderParser = {
+    parse(queryString: string): Order | undefined {
```

Comment:
Have we been adding `undefined` just to be explicit?

Reply:
this function returns undefined so we should be as explicit as possible to help the type system catch errors

Reply:
I guess what I'm saying is that we have strict null type checking off, so I think you don't necessarily need to explicitly add `undefined` but in the event that we turn it back on, this is useful (less work to get everything compiling).

Reply:
I see yes, we should turn that back on at some point, but in either case, the more explicit the better